### PR TITLE
Updated docs to v4.1.0

### PR DESCRIPTION
**Pull request by 416d72**

### CODE REVIEW

**Comment by grischaerbe** [comment]
Good catch!
(I mass replaced the imports based on the directory 🤦‍♂️)
Thanks!

**Review by grischaerbe — CHANGES_REQUESTED** [review]

File: apps/docs/src/routes/extras/use-gltf-animations.md [hunk 1]
```diff
@@ -28,9 +28,7 @@ Convenience hook to use animations loaded with a `<GLTF>` threlte component.
 
 ```svelte
 <script lang="ts">
-  import { GLTF } from '@threlte/extras'
-  import { useGltfAnimations } from 'threlte/extras'
-
+  import { GLTF, useGltfAnimations } from '@threlte/extras'
```
Comment (416d72) [inline apps/docs/src/routes/extras/use-gltf-animations.md:31]: Merged two imports in one statement

File: apps/docs/src/routes/extras/use-gltf-animations.md [hunk 2]
```diff
@@ -56,7 +56,7 @@ Sometimes you might want to use the hook [`useGltf`](/extras/use-gltf) to reuse
 
 ```svelte
 <script lang="ts">
-  import { Object3DInstance } from '@threlte/extras'
+  import { Object3DInstance } from '@threlte/core'
```
Comment (416d72) [inline apps/docs/src/routes/extras/use-gltf-animations.md:57]: importing from `core`

File: apps/docs/src/routes/extras/use-gltf.md [hunk 3]
```diff
@@ -34,7 +34,7 @@ Use the component [`<GLTF>`](/extras/gltf) if you want to use a model in its ent
 
 ```svelte
 <script lang="ts">
-  import { Object3DInstance, Mesh } from '@threlte/extras'
+  import { Object3DInstance, Mesh } from '@threlte/core'
```
Comment (416d72) [inline apps/docs/src/routes/extras/use-gltf.md:37]: importing from `core`

File: packages/extras/src/lib/hooks/useGltfAnimations.ts [hunk 4]
```diff
@@ -11,8 +11,8 @@ import { useFrame } from '@threlte/core'
  *
  * ```svelte
  * <script lang="ts">
- *   import { GLTF } from 'threlte'
- *   import { useGltfAnimations } from '../lib/useAnimations'
+ *   import { GLTF } from '@threlte/core'
```
Comment (grischaerbe) [inline packages/extras/src/lib/hooks/useGltfAnimations.ts:14]: `GLTF` is from /extras, can be merged into one import as well

**Review by grischaerbe — CHANGES_REQUESTED** [review]

File: packages/extras/src/lib/hooks/useGltfAnimations.ts (hunk 4 above)
Comment (416d72) [reply packages/extras/src/lib/hooks/useGltfAnimations.ts:14]: done 👍🏻

**Review by grischaerbe — APPROVED** [review]

**Comment by grischaerbe** [comment]
Thanks so much! <3

**Pull request merged**